Userland/DevTools/HackStudio/Language.cpp

```diff
@@ -30,6 +30,16 @@ Language language_from_file_extension(const String& extension)
     return Language::Unknown;
 }
 
+Language language_from_file_name(const String& name)
+{
+    if (name == "Makefile")
```

Comment:
Maybe we also should add support for the *makefile*s (starting with a lowercase m) and *GNUmakefile*s, as these are also used by gnu make, and bsd make (but without the latter):

> If no -f option is present, make will look for the makefiles GNUmakefile, makefile, and Makefile, in that order.
https://manpages.debian.org/testing/make/make.1.en.html

> By default, the file makefile is used; if no such file is found, it tries Makefile.
https://man.openbsd.org/make

Also, an `.mk` extension can also be a makefile and it's used quite often: https://github.com/search?q=extension%3Amk (12M results)